cost/turbonomics/scale_virtual_machines_recommendations/aws/README.md

```diff
@@ -0,0 +1,22 @@
+# Turbonomics Scale Virtual Machines Recommendations AWS
```

Comment:
Please rename to Turbonomic Rightsize Vitual Machines Recommendations

Reply:
Rename all files!